### DISCUSSION TITLE
Animate crashes too many files

### DISCUSSION THREAD

**Discussion by HanSpyMB** [airsdk/Adobe-Runtime-Support#3252] [General]
Hi,

Animate crashes beyond 28 files and two folders of around fifteen files when publishing an application in Android or iOS.

Have you ever had the problem and found a solution?

THANKS

**Comment by HanSpyMB** [comment]
Animate 24,0,3 build 19
AIR 50.2.4.4

**Reply by ajwfrost** [reply]
Can you try moving your project and files to a short path e.g. "c:\work" to see if that helps? I just added a bunch of files to a project and tried packaging it, it looks like Animate is calling ADT with the full path of all of the files rather than using any of the shortcut mechanisms for picking files from folders. So if you have a lot of files with long paths, you may be hitting the maximum length of a command line...

Try having AIR SDK Manager open with the Troubleshooting tab, and see if anything is shown there (and maybe if not and if it's crashing, try taking some of the 28 files out and re-try, until it works - then check that "ADT xx.x.x.x called with..." line from the the troubleshooting log, and see how longer the command-line was..

thanks

**Comment by HanSpyMB** [comment]
You are amazing, It works.
Thanks thanks thanks ;-)
